tasks/opsep/001-update-l2pao-to-aliased-l1pao/README.md

```diff
@@ -0,0 +1,42 @@
+# Op Sepolia - Update L2ProxyAdmin owner to be aliased L1ProxyAdmin owner address
+
+Status: [DRAFT]()
+
+## Objective
+
+This is the playbook for executing Upgrade 001, to update the L2ProxyAdmin owner (L2PAO) to be the aliased L1ProxyAdmin owner (L1PAO) address, on OP Sepolia (chain ID 11155420).
```

Comment:
[Upgrade 1](https://gov.optimism.io/t/final-upgrade-1-bedrock-protocol-upgrade-v2/5548) is bedrock, this one is part of [Upgrade 8](https://gov.optimism.io/t/final-protocol-upgrade-8-guardian-security-council-threshold-and-l2-proxyadmin-ownership-changes-for-stage-1-decentralization/8157/1)

```suggestion
This is the playbook for executing part of [Upgrade #8](https://gov.optimism.io/t/final-protocol-upgrade-8-guardian-security-council-threshold-and-l2-proxyadmin-ownership-changes-for-stage-1-decentralization/8157/1), to update the L2ProxyAdmin owner (L2PAO) to be the aliased L1ProxyAdmin owner (L1PAO) address, on OP Sepolia (chain ID 11155420).
```

Reply:
Do I then need to change the name of this directory from `001-update-l2pao-to-aliased-l1pao` to `008-update-l2pao-to-aliased-l1pao`?

Reply:
Nope, but I can see how is a bit confusing and something we may want to clarify in the readme:
- The upgrade number being referred to is related to governance, this is the 8th governance proposal
- The 001 prefix on the folder is the ordering of tasks in the opsep director, so this is the first one

(I unresolved for visibility, feel free to resolve when you want)